content-src/styles/variables.scss

```diff
@@ -81,9 +81,13 @@ $search-input-left-label-width: 35px;
 $search-input-border-color: #BFBFBF;
 $search-blue: #0996F8;
 $search-shadow: rgba(96, 181, 255, 0.75);
+$search-box-shadow: rgba(0, 0, 0, 0.1);
```

Comment:
Since this is Sass, you can do `rgba(#000, 0.1)` here, or just use the `$faintest-black` variable from above:

```sass
$faintest-black: rgba($black, 0.1);
```